share/commands/agent-run

```diff
@@ -33,7 +36,10 @@ while getopts "iIvdcb:" opt; do
       ;;
     b)
       BUNDLE="-b ${OPTARG}"
+    q)
+      FILTER="egrep -v @@result_success@@|@@result_na@@|@@log_info@@"
```

Comment:
No it is not.
Pipes are not evaluated once in a variable.
Neither do quotes, so with quotes, it would match the text with quotes included.